### PR TITLE
Mafia: fix html, sort lynches and players

### PR DESCRIPTION
**Pull request by whalemer**

### CODE REVIEW

**Comment by whalemer** [comment]
daily "remove travisbuddy"

**Comment by Zarel** [comment]
Daily "don't push bad code, if it wasn't TravisBuddy yelling at you it'd be me and panpawn"

**Comment by whalemer** [comment]
id fixed it within 30 seconds of pushing it, if it's still wrong after that then i agree that i should be yelled at

File: chat-plugins/mafia.js [hunk 1]
```diff
@@ -1278,9 +1285,12 @@ exports.commands = {
 			let targetUser = this.targetUser;
 			if (!targetUser || !targetUser.connected) return this.errorReply(`The user "${this.targetUsername}" was not found.`);
 			if (!room.users[targetUser.userid]) return this.errorReply(`${targetUser.name} is not in this room, and cannot be hosted.`);
-			if (Object.keys(room.game.players).includes(targetUser.userid) || Object.keys(room.game.dead).includes(targetUser.userid)) return this.errorReply(`You cannot subhost to a user in the game or a user that could be revived.`);
 			if (room.game.hostid === targetUser.userid) return this.errorReply(`${targetUser.name} is already the host.`);
-
+			if (targetUser.userid in room.game.players) return this.errorReply(`You cannot subhost to a user in the game.`);
+			if (targetUser.userid in room.game.dead) {
+				if (cmd !== 'forcesubhost') return this.errorReply(`${targetUser.name} could potentially be revived. To subhost anyway, use /mafia forcesubhost ${target}.`);
+				delete room.game.dead[targetUser.userid];
```
Comment (HoeenCoder) [inline chat-plugins/mafia.js]: You also need to run .destroy() on that before deleting it. (RoomGamePlayer.destroy() to be specific)

**Comment by HoeenCoder** [comment]
Oh, and also update the help info.

File: chat-plugins/mafia.js (hunk 1 above)
Comment (HoeenCoder) [reply chat-plugins/mafia.js]: nvm im dumb

**Comment by whalemer** [comment]
what the fuck github?

**Pull request merged**